speech/grpc/src/main/proto/google/longrunning/operations.proto

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2015, Google Inc.
```

Comment:
This isn't supposed to be in this REPO, and if it is, it shouldn't be in this folder path.

AFAIK, the resolution is not to have this code published.